packages/db-mongodb/src/models/buildSchema.ts

```diff
@@ -52,9 +52,21 @@ type FieldSchemaGenerator = (
   buildSchemaOptions: BuildSchemaOptions,
 ) => void
 
+/**
+ * get a field's defaultValue only if defined and not dynamic so that it can be set on the field schema
+ * @param field
+ */
+const formatDefaultValue = (field: FieldAffectingData) =>
+  typeof field.defaultValue !== 'undefined' &&
+  typeof field.defaultValue !== 'function' &&
+  !field.localized
```

Comment:
couldn't localized fields set their own default values which might be able to be used as well? does this function imply that this won't work if the field is localized?

Reply:
I had to bypass this one for reasons I cannot understand.

>Fields › unique indexes › should not throw validation error saving multiple null values for unique fields
>    	  ValidationError: The following field is invalid: localizedUniqueRequiredText.es